modules/lobbies/utils/captcha_config.ts

```diff
@@ -0,0 +1,9 @@
+import { Config, CaptchaProvider } from "../config.ts";
+
+export const getCaptchaProvider = (config: Config): CaptchaProvider => {
+    if (!config.captcha || !config.captcha.provider) return {
```

Comment:
a null captcha provider currently returns a test provider. we should not enforce captchas if there is no provider provided.